README.md

```diff
@@ -18,7 +18,7 @@ There's a dedicated documentation available at <https://community.algolia.com/an
 
 To contribute to the library (code and documentation), [read our contribution guide](./CONTRIBUTING.md).
 
-[website]: https://community.algolia.com/angular-instantsearch/
+[website]: https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/angular/
```

Comment:
This link reference is not used in the README. We need to use it in the "Documentation" section.